config/Config.js

```diff
@@ -33,14 +33,39 @@ export default {
     addMap: 1,
     removeMap: 1,
   },
+  /**
+   * Enable this for Trakman to take over handling maps. The controller will read maps from a folder
+   * defined below and give the upcoming ones to the server, allowing for much more map capacity than
+   * the server can handle (it becomes basically unresponsive at 5000 maps).
+   */
+  manualMapLoadingEnabled: true,
```

Comment:
i'd put all manual map stuff into its own config category, eg config.manualmaps.enabled etc. also its probably a good idea to disable it by default

Reply:
Agree with the first part, but what's the reason to disable it? It doesn't really make loading maps slower, in fact it's more straight-forward because you can run //udm and all the maps will automatically be added.

Reply:
Ok there are some cases in which it should be disabled, so I'll do that. I will put a recommendation for new servers to enable it on the wiki, though.